src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -99,6 +99,19 @@
  *   &lt;property name=&quot;allowLineBreaks&quot; value=&quot;false&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>To Configure the check to restrict the use of whitespace after TYPECAST:
```

Comment:
Please add same java code example to two config above to show user how behavior changing on same code but different config.

Reply:
Please add java code for each xml config